pkg/random/random.go

```diff
@@ -0,0 +1,20 @@
+package random
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+// Generates a number between min and max inclusive.
+func Range(min, max int) int {
+	if min > max {
+		panic(fmt.Sprintf("min cannot be greater than max: min=%d max=%d", min, max))
```

Comment:
do we recover from this anywhere?

Reply:
No. It's just a sanity check because `rand.Intn`will panic anyway